modules/imap/handler_modules.php

```diff
@@ -836,6 +837,21 @@ public function process() {
                             }
                         }
                     }
```

Comment:
If I'm reading this correctly the failure to find a configured archive folder for the IMAP server results in using a default folder named "Archive", then attempting to move a message to this folder. If that folder does not exist it will fail (or maybe some IMAP servers will auto-create I don't know).

Please remove this default. If the server does not have a configured archive folder just skip it.

Reply:
I have made changes, I remove the previouse behavior which create the Archive folder when the Archive Folder is not defined. Now when the folder is not set and you try to archive mail within the message_list page no feature action will be perform the the action will be skipped.